### PR TITLE
Add Coconut recipe

### PR DESCRIPTION
**Pull request by evhub**
Add recipe for [Coconut](http://coconut-lang.org/) per https://github.com/evhub/coconut/issues/300.

### CODE REVIEW

File: recipes/coconut/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{% set name = "coconut" %}
+{% set version = "1.2.3" %}
+{% set sha256 = "8965cf8d975eebdf567230aa1598cf8efe72dacfcf7fec853dfd09fed16952d7" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
```
Comment (synapticarbors) [inline recipes/coconut/meta.yaml]: Please delete comments.

File: recipes/coconut/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```
Comment (synapticarbors) [inline recipes/coconut/meta.yaml]: It looks like this package uses python entry points. Please add them here:

https://conda.io/docs/building/meta-yaml.html#python-entry-points

File: recipes/coconut/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    - setuptools
+  run:
+    - python
+    - pyparsing
+    - argparse  # [py<27]
+    - pygments  # [py>=27]
+    - prompt_toolkit  # [py>=27]
+    - futures  # [py<3]
+
+test:
```
Comment (synapticarbors) [inline recipes/coconut/meta.yaml]: Please test all command line tools:

https://conda.io/docs/building/meta-yaml.html#test-commands

File: recipes/coconut/meta.yaml (hunk 1 above)
Comment (evhub) [inline recipes/coconut/meta.yaml]: Done.

File: recipes/coconut/meta.yaml (hunk 2 above)
Comment (evhub) [inline recipes/coconut/meta.yaml]: Done.

File: recipes/coconut/meta.yaml (hunk 3 above)
Comment (evhub) [inline recipes/coconut/meta.yaml]: Done.

**Pull request merged**